src/Rules/Program.hs

```diff
@@ -37,8 +37,26 @@ buildProgram rs context@Context {..} = when (isProgram package) $ do
         -- We build only unwrapped binaries in inplace/lib/bin
         inplaceLibBinPath -/- programName context <.> exe %> \bin -> do
             binStage <- installStage
+            buildBinaryAndWrapperLib rs (context { stage = binStage }) bin
+
+        inplaceLibBinPath -/- programName context <.> "bin" <.> exe %> \bin -> do
```

Comment:
But wait, since it is not built at all, so that doesn't harm right? What if Windows support gets added someday in future? I still think that leaving the code as general as we can now would be beneficial.